docs/color_system.md

```diff
@@ -0,0 +1,71 @@
+
+## The colors
+
+There are 10 base colors specified in the Textual Color System. Although it is unlikely that all will need to be specified, since some may be derived from others, and some defaults may not need to be changed.
+
+### Shades
+
+Each color has 6 additional shades (3 darker and 3 lighter), giving a total of 7 shades per color.
+
+### Primary and Secondary
+
+The _primary_ and _secondary_ colors are used as a background for large areas of the interface, such as headers and sidebars. The secondary color is optional, and if not supplied will be set to be the same as primary. If supplied, the secondary color should be compliment the primary, and together can be considered the _branding colors_ as they have the greatest influence over the look of the TUI.
+
+### Background and Surface
+
+The _surface_ colors is the base color which goes behind text. The _background_ color is typically the negative space where there is no content.
+
+These two colors tend to be very similar, with just enough difference in lightness to tell them apart. They should be chosen for good contrast with light or dark, depending on the
+
+In light mode the default background is #efefef (a bright grey) and the surface is #f5f5f5 (off white). In dark mode the default background is 100% black, and the default surface is #121212 (very dark grey).
+
+Note that, although both background and surface support the full range of shades, because they are close to the extremes, it map not be possible to darken or lighten them further. i.e. you can't get any lighter than 100% white or darken than 100% black.
+
+### Panel
+
+The _panel_ color is typically used as a background to emphasize text on the default surface, or as a UI element that is above the regular UI, such as a menu.
+
+The default panel color is derived from the surface color by blending it towards either white or black text (depending on mode).
+
+Unlike background and surface, the panel color is automatically selected so that it can always be lightened or darkened by the full degree.
+
+### Accent
+
+The _accent_ color should be a contrasting color use in UI elements that should stand out, such as selections, status bars, and underlines.
+
+### Warning, Error, and Success
+
+The _warning_, _error_, and _success_ colors have semantic meaning. While they could be any color, by convention warning should be amber / orange, error should be red, and success should be green.
+
+### System
+
+The system color is used for system controls such as scrollbars. The default is for the system color to be the same as accent, but it is recommended that a different color is chosen to differentiate app controls from those rendered by the Textual system.
```

Comment:
> but it is recommended that a different color is chosen to differentiate app controls from those rendered by the Textual system.

IMO this recommendation doesn't solve a real problem. More consistency in colours is more important to me than "differentiating app controls from those rendered by Textual". Knowing whether something on screen is rendered by the app or the app framework probably won't matter to end users.

Reply:
Perhaps. It will default to accent if it is not set. Having a different scrollbar color does help when your status bar is the accent color.